server/utils/fields.py

```diff
@@ -249,12 +258,15 @@ def list_resolver(
         count = qs.count()
 
         if getattr(self, "pagination", None):
+            ordering = kwargs.pop(self.pagination.ordering_param, None) or self.pagination.ordering
+            ordering = ','.join([to_snake_case(each) for each in ordering.strip(',').replace(' ', '').split(',')])
+            self.pagination.ordering = ordering
             qs = self.pagination.paginate_queryset(qs, **kwargs)
 
         return CustomDjangoListObjectBase(
             count=count,
             results=maybe_queryset(qs),
             results_field_name=self.type._meta.results_field_name,
-            page=kwargs.get('page', 1),
-            pageSize=kwargs.get('pageSize', graphql_api_settings.DEFAULT_PAGE_SIZE)
+            page=kwargs.get('page', 1) if 'page' in kwargs else None,
```

Comment:
This should be fine right?
```suggestion
            page=kwargs.get('page'),
```